MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+
+* Allows character set of digits (0-9), upper case alphabets (A-Z), and symbols like space, minus (-), plus (+), period (.), dollar sign ($), slash (/), and percent (%).
+* Each character is encoded with 5 bars and 4 spaces where 3 are wide and 6 are narrow.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="CODE39"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:Code39 Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "CODE39";
+    barcode.Symbology = new Code39() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code39 Symbology](images/one-dimensional-symbology/maui-code39-symbology.png)
+
+The `EnableCheckSum` property of `Code39` barcode allows to add the check digit along with the input value. The default value of `EnableCheckSum` property is true.
+
+## Code39 Extended
+
+* `Code39 Extended` symbology is an extended version of Code39 that supports all 128 ASCII characters set. So, it encodes lower case alphabets (a-z) as well as special characters.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="051091"
+                               ShowText="True"
+                               HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:Code39Extended Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "051091";
+    barcode.Symbology = new Code39Extended() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code39 Extended Symbology](images/one-dimensional-symbology/maui-code39-extended-symbology.png)
+
+As like `Code39`, the `Code39 Extended` supports the `EnableCheckSum` property.
+
+## Code93
+
+`Code93` is designed to complement and enhance `Code39`. It represents the complete ASCII character set by using a combination of 2 characters. It is a continuous, variable-length symbology and produces a denser code.
```

Comment:
It is a continuous, variable-length symbology that produces a denser code.

Reply:
changes updated.